src/scss/includes/direction-form.scss

```diff
@@ -46,20 +48,18 @@
       margin: 0 52px 0 44px;
     }
 
-    .direction-invert-origin-destination {
-      position: absolute;
+    .direction-invert-button {
       display: flex;
       align-items: center;
       justify-content: center;
-      width: 36px;
-      height: 36px;
-      right: 8px;
-      top: 22px;
-      cursor: pointer;
+      min-width: 36px;
```

Comment:
Why these `min-*`? It looks like it's allowed grow, whereas it shouldn't. Maybe what you want is `flex-grow: 0` and `flex-shrink: 1`?